deploy/configs/applications/modules.yaml

```diff
@@ -39,8 +39,7 @@ modules:
       - functionalizer
       - meshball
       - nest
-      - 'neurodamus%intel'
-      - 'neurodamus-core+common'
+      - neurodamus-core
```

Comment:
to clarify: Are we deploying +common? If so, Isn't `+common` better to be sure that neurodamus-core with +common is deployed?

Reply:
`+common` is deployed just because of bglibpy and Werner case. In the generic case people will be loading a pure core and the cell mechanisms they want, including their own common. That 's the case of the pre-build model-neocortex, and others to come.

Reply:
Importantly, `core+common` cannot be used together with `model-neurodamus` as both would bring the common (AMPANMDA, gap, etc)

Reply:
@ferdonline: ok...what I meant was can we add +common or -common there in the module name so that it’s clear.